cgi/product_multilingual.pl

```diff
@@ -2152,14 +2152,17 @@ ($$$$$$)
 ;
 
 	if ($type eq 'edit') {
-		$html .= <<HTML
+		$html .= <<"HTML"
```

Comment:
What do the quotes around the Heredoc cause?

Reply:
See https://github.com/openfoodfacts/openfoodfacts-server/pull/2018#discussion_r298182422